### PR TITLE
chore: Update readme file

### PR DESCRIPTION
**Pull request by AfsarFresh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,108 @@
...
+
+## Project Resources
+---
+
+- [JIRA Board](https://freshworks.atlassian.net/)
+- [Designs](https://www.figma.com/)
+- [Project Overview](https://sites.google.com/freshworks.io/project-hub/projects)
+- [App Store](https://apps.apple.com/ml/app/yukon-vaccination-verifier/id1591117176)
+
+## Architecture
```
Comment (freshfawzy) [inline README.md:31]: I think we should use another name, to differ between this and the sub-bullets,  maybe "System Design" ?

**Review by freshfawzy — COMMENTED** [review]
It looks great, but suggesting one change, please check it out

File: README.md (hunk 1 above)
Comment (AfsarFresh) [reply README.md:31]: Please check now.

**Review by freshfawzy — APPROVED** [review]
It looks great, thanks!

**Pull request merged**